src/index.js

```diff
@@ -0,0 +1,30 @@
+import './style.css';
+import {
+  getStorageData, clearCompletedList, clearCheckBox, addTodo, saveEdit, displayToDo, getIsEditing,
+} from './module/addRemove.js';
+
+window.onload = () => {
+  getStorageData();
+  displayToDo();
+};
+
+const refreshBtn = document.querySelector('.refresh-btn');
+refreshBtn.addEventListener('click', () => {
+  clearCheckBox();
+  window.location.reload();
+});
+
+const descriptions = document.querySelector('.inputTask');
+descriptions.addEventListener('keyup', (event) => {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    if (!getIsEditing()) {
+      addTodo();
+    } else saveEdit();
```

Comment:
- [ ] Please add curly braces around the else statement for `saveEdit() `
      To improve code clarity and consistency. :+1:

Reply:
Thank you buddy! It is done :+1: